doc/source/user_guide/groupby.rst

```diff
@@ -274,15 +274,16 @@ the length of the ``groups`` dict, so it is largely just a convenience:
    df
    gb = df.groupby("gender")
 
-
-.. ipython::
+.. ipython:: python
 
    @verbatim
-   In [1]: gb.<TAB>  # noqa: E225, E999
+   :okexcept:
```

Comment:
Why did this need to change?

Reply:
The ``gb.<TAB>`` doesn't pass the tests when I try to build the documentation. I also added ``:okexcept:`` because of a build error.

Reply:
Sure!

Reply:
Looks like these changes still exist in the latest commit